Libraries/Inspector/ElementProperties.js

```diff
@@ -24,29 +23,21 @@ const flattenStyle = require('flattenStyle');
 const mapWithSeparator = require('mapWithSeparator');
 const openFileInEditor = require('openFileInEditor');
 
-import type {DangerouslyImpreciseStyleProp} from 'StyleSheet';
+import type {ViewStyleProp} from 'StyleSheet';
 
-class ElementProperties extends React.Component<{
+type Props = $ReadOnly<{|
   hierarchy: Array<$FlowFixMe>,
-  style?: DangerouslyImpreciseStyleProp,
-  source?: {
-    fileName?: string,
-    lineNumber?: number,
+  style?: ?ViewStyleProp,
+  source?: ?{
+    fileName?: ?string,
```

Comment:
I'm not sure these should be nullable. Lets leave fileName and lineNumber the way they were:

```
fileName?: string,
lineNumber?: string
```